test/gurka/test_admin.cc

```diff
@@ -10,7 +10,7 @@ using namespace valhalla;
 const std::unordered_map<std::string, std::string> build_config{
     {"mjolnir.data_processing.use_admin_db", "false"}};
 
-class Admin : public ::testing::Test {
+class AdminTest : public ::testing::Test {
```

Comment:
I had name-collisions here, likely due to the include of `test.h` from `gurka.h` so renamed.